examples/plot_simulate_evoked.py

```diff
@@ -35,7 +35,8 @@
 
 ###############################################################################
 # This is a lot of parameters! We can also filter the
-# parameters using unix-style wildcard characters
+# parameters using unix-style wildcard characters. Most of the parameters
+# relate to within-network connections and the cell geometry.
 print(params['L2Pyr_soma*'])
```

Comment:
Since we're discouraging modifications to `params`, is this counterproductive without an API to interact with these network properties?

Reply:
I see your point @ntolley , but I guess I can see the value in printing them here. Your call @jasmainak

Reply:
I removed the scaling factor for now. I think those functions need to be cleaned up before they can be exposed ... and also removed the units. Let's fix it in a separate PR

Reply:
oops, this was meant as a reply to [this comment](https://github.com/jonescompneurolab/hnn-core/pull/254#discussion_r564753806)